client/ayon_resolve/plugins/create/create_shot_clip.py

```diff
@@ -223,7 +339,25 @@ def create(self, subset_name, instance_data, pre_create_data):
                 "Go into a timeline and then reset the publisher."
             )
 
-        self.log.debug(f"Selected: {self.selected}")
+        if not self.selected:
+            if pre_create_data.get("use_selection", False):
+                raise CreatorError(
+                    "No Chocolate-colored clips found from "
+                    "timeline.\n\n Try changing clip(s) color "
```

Comment:
```suggestion
                    "timeline.\n\nTry changing clip(s) color "
```

Fix space on second line.
![image](https://github.com/user-attachments/assets/b0888b06-a608-4643-a956-318254dc72a7)

After:
![image](https://github.com/user-attachments/assets/90625036-649e-4f04-9ff8-0242d57b9dcb)